contrib/depends/packages/packages.mk

```diff
@@ -1,18 +1,22 @@
-ifeq ($(host_os),android)
 packages:=boost openssl zeromq libiconv
-else
-packages:=boost openssl zeromq expat ldns libiconv hidapi protobuf libusb
```

Comment:
expat and ldns have gone, was that intended ?

Reply:
Nothing was linking to them, that I could see. No references in CMakeLists.txt anywhere either. Did I miss something?

Reply:
If you did this on purpose, all good. I just read the diff and saw the removal. All good then.